eng/common/templates/job/build-from-source.yml

```diff
@@ -0,0 +1,75 @@
+parameters:
+  jobNamePrefix: 'sourcebuild'    # Override for prefix for job names
+  managedOnly: false              # Specifies whether source is managed only. If so, one build platform is sufficient
+  portableOnly: false             # If true, builds one leg with portable flag enabled
+  prebuiltCheck: true             # Specifies whether to enforce prebuilt checking and fail if new prebuilts are introduced
```

Comment:
Put comments on line before parameter? Matches the existing Arcade SDK template style, more maintainable (alignment!), and IMO more readable.

https://github.com/dotnet/source-build-reference-packages/blob/50dfd943f6e188f47fe138c1c43f2832e1b09555/eng/common/templates/jobs/jobs.yml#L1-L15

Reply:
I would argue that there's not an existing Arcade SDK template style.  They're a bit of a mixed bag.  But I'm fine with changing it.

Reply:
Huh, yeah. I looked at a few important-looking once, but just happened to match.